blog/2023-07-31-new-pre-post-command-plugin-hooks.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: "New pre and post command plugin hooks"
+slug: "2023-07-31-latest-conda-release-includes-new-plugin-hooks"
+description: |
+  The latest release of conda introduces two new plugin hooks: pre and post command. These hooks allow plugin
+  authors to execute code before and after any conda command. Learn more about how and why you would use
+  these in this post.
+authors: [travishathaway]
+tags: [conda, plugins]
+image: img/blog/2023-07-31-new-pre-post-command-plugin-hooks/pre-post-command-image.jpg
+---
+
+[conda-issue]: https://github.com/conda/conda/issues/12138
+[conda-guard]: https://github.com/travishathaway/conda-guard
```

Comment:
Hmm, hmm, given the potential use of the name elsewhere, could we not mention that?

Reply:
I understand the concern. I really want to link to an example project though because I think it will enhance the experience.

Maybe a new name is warranted? What about `conda schutz`? 😂 🇩🇪. It's German so there should be any naming conflicts. Plus, it's short and easy to type.

I'll keep trying to find other names for the project...

Reply:
I'm going to go with "schutz" 😄.

Reply:
Okay, I went with "conda-protect" instead.

Reply:
OMG, only see this now! "schutz" has some unfortunate historic connotations, let's try to prevent that :D